compiler_opt/es/blackbox_learner_test.py

```diff
@@ -115,12 +116,15 @@ def setUp(self):
                                     'policy')
     saver.save(policy_save_path)
 
-    self._saved_policies = []
+    self._iteration_policies_path = os.path.join(output_dir.full_path,
+                                                 'policies')
+    os.mkdir(self._iteration_policies_path)
```

Comment:
`exist_ok=True`?

Reply:
Added an assertion. It should not exist given it's unique per test.